mimic/rest/maas_api.py

```diff
@@ -345,7 +330,7 @@ def list_entities(self, request, tenant_id):
         metadata['next_marker'] = None
         metadata['next_href'] = None
         request.setResponseCode(200)
-        return json.dumps({'metadata': metadata, 'values': entities})
+        return json.dumps({'metadata': metadata, 'values': entities}).encode('utf-8')
```

Comment:
I am pretty sure that this is unnecessary, since `json.dumps` is already returning a `str` object on py2 (which is equivalent to a `bytes` object).  In fact, it's technically wrong, because you are encoding already-encoded data and run the risk of encoding errors if there's any non-ASCII.  On py3 it's a `str` but we're not worrying about that just yet :).